share/spice/currency/currency.css

```diff
@@ -205,6 +205,12 @@
     }
 }
 
+@media only screen and (max-width: 600px) {
```

Comment:
Looks like this is a little too restrictive. LandscapeiPhone 6Plus is 700+ px wide:

![299_usd_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/14123280/cb020028-f5cd-11e5-8b90-96c0df74f286.png)

It may be better to use our `is_mobile` js variable to set a mobile class that sets a width on the image, regardless of the device width.

Alternatively you can try targeting `.is-mobile .flag_img` -- we add an `is-mobile` class to `<html>` on mobile devices :+1: